lib/ui/hooks.dart

```diff
@@ -4,6 +4,28 @@
 
 part of dart.ui;
 
+@pragma('vm:entry-point')
+void _updateDisplays(
+  List<int> ids,
+  List<double> sizes,
```

Comment:
What do you think of using different parameters for widths/heights? It makes the size logic a bit simpler. I don't feel strongly about this

Reply:
Done